### DISCUSSION TITLE
Faster way to move documents?

### DISCUSSION THREAD

**Discussion by designatedsuccessor** [outline/outline#7371] [General]
We have about 100 documents.  We decided we wanted to organize them into nested documents, so we created about 5 "top level" documents.  Then, we started moving documents into one of the top-level ones.  This process was very slow and manual, and it took several hours.

Is there a better way to do this than one-by-one?  Is there a way to select, say, 10 documents and then move them all at once?

**Comment by tommoor** [comment]
I would say right now there is not a better way, but I'm surprised this hasn't been requested more up until now – there certainly needs to be some ability to select and "bulk move" documents around.
